ni/panels/v1/panel_service.proto

```diff
@@ -0,0 +1,146 @@
+option java_package = "com.ni.panels.v1";
+option objc_class_prefix = "NIPS";
+option php_namespace = "NI\\Panels\\V1";
+option ruby_package = "NI::Panels::V1";
+
+// Service interface for interacting with NI panels
+service PanelService {
+  // Start a panel using the provided configuration (or connect to if it has already been started)
+  // Status Codes for errors:
+  // - INVALID_ARGUMENT: 
+  //   - The panel identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  //   - The panel configuration has an invalid argument.
+  // - NOT_FOUND: 
+  //   - The panel configuration includes a file that was not found.
+  rpc StartPanel(StartPanelRequest) returns (StartPanelResponse);
+
+  // Stop a panel
+  // Status Codes for errors:
+  // - INVALID_ARGUMENT:
+  //   - The panel identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  rpc StopPanel(StopPanelRequest) returns (StopPanelResponse);
+
+  // Enumerate the panels available in the system, including information about the state of the panels and what values they have.
+  // Status Codes for errors:
+  rpc EnumeratePanels(EnumeratePanelsRequest) returns (EnumeratePanelsResponse);
+
+  // Get a value for a control on the panel
+  // Status Codes for errors:
+  // - INVALID_ARGUMENT: 
+  //   - The panel identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  //   - The value identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  // - NOT_FOUND: 
+  //   - The value with the specified identifier was not found.
+  rpc GetValue(GetValueRequest) returns (GetValueResponse);
+
+  // Try to get a value for a control on the panel
+  // Status Codes for errors:
+  // - INVALID_ARGUMENT: 
+  //   - The panel identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  //   - The value identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  rpc TryGetValue(TryGetValueRequest) returns (TryGetValueResponse);
+
+  // Set a value for a control on the panel
+  // Status Codes for errors:
+  // - INVALID_ARGUMENT: 
+  //   - The panel identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  //   - The value identifier contains invalid characters. Only alphanumeric characters and underscores are allowed.
+  rpc SetValue(SetValueRequest) returns (SetValueResponse);
+}
+
+message StartPanelRequest {
+  // Unique ID of the panel
+  string panel_id = 1;
+
+  // Configuration for the panel, packed as Any
+  google.protobuf.Any panel_configuration = 2;
+}
+
+message StartPanelResponse {
+  // Location of the panel 
+  string panel_uri = 1; 
+}
+
+message StopPanelRequest {
+  // Unique ID of the panel
+  string panel_id = 1;
+
+  // Reset all storage associated with panel
+  bool reset = 2;
```

Comment:
I guess this means all of the values and `value_ids` associated with the panel are cleared such an attempt to get any value from the panel will fail? Does this have any broadcast/notification associated with it similar to `SetValueRequest`?